translate/rus.json

```diff
@@ -220,7 +220,7 @@
 "live pv":"живой ов",
 "live":"живой",
 "live~2":"прямой эфир",
-"livelog":"живой журнал",
+"livelog":"log-журнал",
```

Comment:
This change you forgot to transfer from my last patch. Check here: https://github.com/TCEC-Chess/tcecgui/pull/192